.buildkite/test-template.j2

```diff
@@ -36,7 +37,7 @@ steps:
     agents:
       queue: neuron
     command: bash .buildkite/run-neuron-test.sh
-    soft_fail: true
+    soft_fail: false
```

Comment:
Should the neuron test be kept as soft_fail as well?

Reply:
This is intentional because it has been stable for quite a while, and proving a point that we will eventually flip AMD back.